src/main/resources/api.yaml

```diff
@@ -1136,6 +1188,42 @@ components:
           type: string
         otherProperties:
           type: string
+    CommentDto:
+      type: object
+      required:
+        - commentId
+        - ownerId
```

Comment:
Maybe instead of just `id` of the comment author we could return whole `UserSummaryDto` to display displayName.
Or just `displayName` instead of `id`? (Imo we could return user with id, email, displayName, but I leave decision to you)